src/Google.Cloud.Functions.Invoker.Testing/FunctionTestServer.cs

```diff
@@ -65,6 +70,28 @@ public FunctionTestServer(Type functionType)
         /// <returns>An <see cref="HttpClient"/> that will connect to <see cref="Server"/> by default.</returns>
         public HttpClient CreateClient() => Server.CreateClient();
 
+        /// <summary>
+        /// Clears all logs.
+        /// </summary>
+        public void ClearLogs() => _testLoggerProvider.Clear();
+
+        /// <summary>
+        /// Returns a list of log entries for the given category name. If no logs have been
+        /// written for the given category, an empty list is returned.
+        /// </summary>
+        /// <param name="type">The type from which to derive the category name.</param>
+        /// <returns>A list of log entries for the given category name.</returns>
+        public List<TestLogEntry> GetLogEntries(Type type) => GetLogEntries(type.FullName ?? "");
```

Comment:
```suggestion
        public List<TestLogEntry> GetLogEntries(Type type) => GetLogEntries(type?.FullName ?? "");
```
(I hope ?? doesn't "translate" NullReferenceException to null :))

Reply:
Have made it a precondition instead - and below for the category name.